### DISCUSSION TITLE
What advantage does a basic merge have and how is it different from a compressed merge?

### DISCUSSION THREAD

**Discussion by A-Really-Big-Cat** [bcssov/IronyModManager#276] [Q&A]
Can a basic merge also have performance benefits?

**Comment by bcssov** [comment]
It's covered in the wiki (or pdf docs) and there's a discussion here that also covers some aspects of both. You can search compress it should yield some results. 
In game aspects compress is superior as game has less file handles open so it give superior game performance.
